src/modules/back-to-top/back-to-top.component.spec.ts

```diff
@@ -53,12 +53,12 @@ describe('StacheBackToTopComponent', () => {
 
   it('should show when the window y offset is greater than the specified offset', fakeAsync(() => {
     component.offset = 0;
-    TestUtility.fireDomEvent(windowRef, 'scroll');
+    TestUtility.triggerDomEvent(windowRef, 'scroll');
```

Comment:
"Trigger" is more often used than "fire". [For example...](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events)